packages/apps/graph-client/src/pages/account/overview/[module]/[account]/[chain].tsx

```diff
@@ -124,9 +125,14 @@ const ChainAccount: React.FC = () => {
                 </Grid.Item>
                 <Grid.Item>
                   <CompactTransactionsTable
-                    moduleName={router.query.module as string}
-                    accountName={router.query.account as string}
-                    chainId={router.query.chain as string}
+                    // moduleName={router.query.module as string}
+                    // accountName={router.query.account as string}
+                    // chainId={router.query.chain as string}
+                    viewAllHref={`${routes.ACCOUNT_TRANSACTIONS}/${
+                      router.query.module
+                    }/${
+                      router.query.account
+                    }${`?chainId=${router.query.chain}`}`}
```

Comment:
In this particular page I think it will, since for us to be able to get the `useGetChainAccountQuery` we need to have `chainId` as an argument ( it is required ).
For the `packages/apps/graph-client/src/pages/account/overview/[module]/[account].tsx` I might need to make the change however.

Reply:
I will add the condition nonetheless. The router param has undefined as a possible type, so I will add the check